### ISSUE TITLE
[v3.2.0] Why removing the property dct:type from the class cv:Event?

### ISSUE THREAD

**Issue by jimjyang** [SEMICeu/CPSV-AP#126]
I am wondering why removing the property dct:type from the [class cv:Event](https://semiceu.github.io/CPSV-AP/releases/3.2.0/#Event), when both its subclasses cv:LifeEvent and cv:BusinessEvent have dct:type? Which specific rule in the SEMIC Style Guide this potentially will be in conflict with? Is it "forbidden" to inherit a property and recommend usage of a given controlled vocabulary for that property at the subclass level?

**Comment by EmielDh** [comment]
I agree with Jim Yang on this. In addition I would propose to, at a SEMIC level introduce a syntax, to indicate whether an attribute was inherited. If the fear is that it would not be clear that the class was inherited and clarified this could give an initial indication.
For example in the VL CPV the properties originating from foaf:Person get a little marker versus CPV where it is not clear at a glance: 
VL:
<img width="176" height="165" alt="Image" src="https://github.com/user-attachments/assets/4b3d6e6b-657e-4115-890a-f25cf6c0d1dc" />
EU:
<img width="217" height="217" alt="Image" src="https://github.com/user-attachments/assets/2401693e-b796-45ad-8a83-0db39e84383f" />
